modules/cli-installing-cli.adoc

```diff
@@ -82,12 +81,18 @@ ifdef::openshift-origin[]
 . Navigate to link:https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the folder for your operating system and architecture.
 . Download `oc.tar.gz`.
 endif::[]
-ifndef::openshift-origin[]
+ifndef::openshift-origin,microshift[]
 . Navigate to the link:https://access.redhat.com/downloads/content/290[{product-title} downloads page] on the Red Hat Customer Portal.
 . Select the architecture in the *Product Variant* drop-down menu.
 . Select the appropriate version in the *Version* drop-down menu.
 . Click *Download Now* next to the *OpenShift v{product-version} Linux Client* entry and save the file.
 endif::[]
+ifdef::microshift[]
+. Navigate to the link:https://access.redhat.com/downloads/content/290[{ocp} downloads page] on the Red Hat Customer Portal.
+. Select the architecture in the *Product Variant* drop-down menu.
```

Comment:
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#d
Use with caution "drop-down"   adjective
Use only to describe a GUI component, followed by _menu_ or _list._
--Per the above guidance, "menu" seems acceptable; however if you look later at the UI elements section, these items do look more like a list, https://www.ibm.com/docs/en/ibm-style?topic=elements-ui
--Note that these other lines (135, 136, 178, 179) are not part of what was changed with this PR
--This is an interesting edit since this module is used all over the place and likely predates the merger with IBM. I'm going to reach out to the CCS style folks before applying a change. Stay tuned!

Reply:
@sjhala-ccs The guidance was both are correct; however, I agree that these look more like lists since there are no sub-items populated when you select product or arch. It is out of scope, but also an easy fix, so I did update these to "from" and "list."